draft-ietf-rtcweb-jsep.xml

```diff
@@ -4076,6 +4076,65 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             process, as a remote offer or answer, unmodified SDP coming
             from any other implementation of this specification.</t>
         </section>
+        <section title="Processing RTP packets"
+                 anchor="sec.rtp.demux">
+          <t>When an RTP packet is received by a transport (typically a
+          DtlsTransport), that packet MUST be routed to the correct
+          RtpReceiver. For each transport, the following steps MUST be
+          followed to prepare to route packets:</t>
+
+          <t>
+            <list>
+              <t>Construct a table mapping MID to RtpReceiver. For each
+              RtpReceiver configured to recei ve from this transport,
+              let the key be the RtpReceiver's MID, and the value be the
+              RtpReceiver.</t>
+              
+              <t>Construct a table mapping SSRC to RtpReceiver. For each
```

Comment:
transceiver.receiver.getParameters().encodings[j].ssrc, transceiver.receiver.getParameters().encodings[j].rtx.ssrc, and transceiver.receiver.getParameters().encodings[j].fec.ssrc

are all defined in the W3C doc, and those values are ultimately filled in by the remote description, which is defined in this document.  So shouldn't we be able to just say "via the SSRCs found in the remote description"?